src/Filesystem.cc

```diff
@@ -220,7 +220,7 @@ namespace gz
 #endif  // _WIN32
 
     //////////////////////////////////////////////////
-    const std::string separator(const std::string &_p)
```

Comment:
return type 'const std::string' (aka 'const basic_string<char>') is 'const'-qualified at the top level, which may reduce code readability without improving const correctness